code/controllers/subsystems/ticker.dm

```diff
@@ -453,7 +453,7 @@ var/datum/controller/subsystem/ticker/SSticker
 			available_ghostroles |= G.name
 
 	// Special case, to list the Merchant in case it is available at roundstart
-	if(SSjobs.type_occupations[/datum/job/merchant].total_positions)
+	if(SSjobs.type_occupations[/datum/job/merchant]?.total_positions)
```

Comment:
why is this being changed?

Reply:
this causes a runtime when the merchant job isn't one of the available jobs on, say, an event map

Reply:
thanks for the explanation!